vulnfeeds/cmd/cperepos/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:alpine@sha256:74a382917f6eaa7cc2d000dc2cd412a7f823f343b3b6268b20d84d057bc56718
+
+RUN mkdir /src
+WORKDIR /src
+
+COPY ./go.mod /src/go.mod
+COPY ./go.sum /src/go.sum
+RUN go mod download
+
+COPY ./ /src/
+RUN go build -o cperepos ./cmd/cperepos
+
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine
+
+ENTRYPOINT ["/src/cmd/cperepos/gen_cperepos_map"]
```

Comment:
Are we missing a `COPY --from=` here ?

Reply:
I may have gotten carried away with my multi-stage build conversion, [3651384](https://github.com/google/osv.dev/pull/1146/commits/36513842d83d7d4980cbfc539dae13f7337d35c6) ought to fix it

Reply:
Oh I see what I've done wrong, fixed (hopfully?) in [e16864f](https://github.com/google/osv.dev/pull/1146/commits/e16864f722733227c104c0bc5d7d3d254036edc1)